### PR TITLE
New prototype version (0.92) with breaking changes

### PR DESCRIPTION
**Pull request by perwestling**
This will probably break all 18SJ games; so just as well pin them all.

Changes in this commit that now corresponds to version 0.92:
- Update terrain cost
- Update terrain cost for Gelivare private company
- KHJ train not allowed to be used twice in an OR
- Bergslagen bonus no longer possible for Malmö
- Major redesign of Göta Kanalbolaget
- Bonus table has been moved from Route dialog to map

This version has been play tested.

### CODE REVIEW

**Review by ventusignis — COMMENTED** [review]
Is it possible to add a hotseat spec to replace the one that's getting removed here?

**Comment by perwestling** [comment]
> Is it possible to add a hotseat spec to replace the one that's getting removed here?

It is possible. Have a play test that run until phase 5, which I think I can clone and then complete. And use as a spec.

File: lib/engine/game/g_18_sj/game.rb [hunk 1]
```diff
@@ -333,7 +349,7 @@ def select(collection)
         end
 
         def find_company(companies, collection)
-          sym = collection[rand % collection.size]
+          sym = select(collection)
```
[the remark this answers is not in the document]
Comment (perwestling) [reply lib/engine/game/g_18_sj/game.rb:352]: No, it is a method defined at line 347-349 so it should return a random element from the collection.

**Comment by perwestling** [comment]
@ventusignis I will add the task of adding a spec file to my backlog for 18SJ. And I will fix the log message you spotted. Thanks for checking my review.

**Pull request merged**